test/functional/api/vim_spec.lua

```diff
@@ -177,6 +178,62 @@ describe('vim_* functions', function()
     end)
   end)
 
+  describe('err_write', function()
+    before_each(function()
+      clear()
+      screen = Screen.new(40, 8)
+      screen:attach()
+      screen:set_default_attr_ids({
+        [1] = {foreground = Screen.colors.White, background = Screen.colors.Red},
+        [2] = {bold = true, foreground = Screen.colors.SeaGreen}
+      })
+      screen:set_default_attr_ignore( {{bold=true, foreground=Screen.colors.Blue}} )
+    end)
+
+    it('can show one line', function()
+      nvim_async('err_write', 'has bork\n')
```

Comment:
@bfredl  What if `nvim_async('err_write, ...)` is called multiple times? Do we need a test for that? E.g.:

```
nvim_async('err_write', 'has bork\n')
nvim_async('err_write', 'has bork\n')
nvim_async('err_write', 'has bork\n')
```

What will the result be?